### PR TITLE
Login page updated

### PR DESCRIPTION
**Pull request by Srilaya-sriraman**

### CODE REVIEW

File: Client/src/Screens/App.css [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#root {
```
Comment (poseidon0z) [inline Client/src/Screens/App.css:1]: delete all this default code

File: Client/src/Components/Google.jsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
+function Google(){
```
Comment (poseidon0z) [inline Client/src/Components/Google.jsx:1]: delete this component, use the icon as an image instead

File: Client/src/App.jsx [hunk 3]
```diff
@@ -1,5 +1,16 @@
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
```
Comment (poseidon0z) [inline Client/src/App.jsx:1]: where did all the routes go?

**Review by poseidon0z — CHANGES_REQUESTED** [review]
Please fix requested changes in comments

File: Client/src/Components/Google.jsx (hunk 2 above)
Comment (Sushindh) [reply Client/src/Components/Google.jsx:1]: but i can't get this icon image from figma
it is available as svg links
can i use from google images

File: Client/src/Screens/App.css (hunk 1 above)
Comment (Sushindh) [reply Client/src/Screens/App.css:1]: ok

File: Client/src/App.jsx (hunk 3 above)
Comment (Sushindh) [reply Client/src/App.jsx:1]: the routes for routing signup page to login and routing login page to signup page

File: Client/src/Components/Google.jsx (hunk 2 above)
Comment (poseidon0z) [reply Client/src/Components/Google.jsx:1]: yes you can use google images

File: Client/src/App.jsx (hunk 3 above)
Comment (poseidon0z) [reply Client/src/App.jsx:1]: yeah there's none of them in this screen though

**Review by poseidon0z — APPROVED** [review]

**Pull request merged**